main.go

```diff
@@ -1,16 +1,59 @@
 package main
 
 import (
+	"context"
 	"net"
+	"time"
 
 	service "github.com/Merteg/pl-health-service/pkg/service"
 	"github.com/Merteg/pl-health-service/proto"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"gopkg.in/mgo.v2/bson"
 
 	"github.com/rs/zerolog/log"
 	"google.golang.org/grpc"
 )
 
 const port string = "localhost:8080"
+const mongoURI string = "mongodb+srv://admin:admin@pl-health-service.s25udti.mongodb.net/test"
+
+func init() {
+	client, err := mongo.NewClient(options.Client().ApplyURI(mongoURI))
+	if err != nil {
+		log.Fatal().Err(err)
+	}
+	ctx, _ := context.WithTimeout(context.Background(), 10*time.Second)
+	err = client.Connect(ctx)
+	if err != nil {
+		log.Fatal().Err(err)
+	}
+	err = client.Ping(ctx, nil)
+	if err != nil {
+		log.Fatal().Err(err)
+	}
+
+	targetCollection, err := client.Database("pl-health-service").ListCollectionNames(context.TODO(), bson.M{"name": "target"})
```

Comment:
Create a constant for database name pls
Also for collection name
Note that collection names should be 'targets' and 'health'